tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_packet_threshold_size/run.py

```diff
@@ -0,0 +1,79 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate message size that is published on tedge/measurements,
+by subscribing for error message on tedge/errors by tedge-mapper c8y
+
+Given a configured system
+When we start the tedge-mapper-c8y systemctl service
+When we start tedge sub with sudo in the background for error messages
+When we publish a bigger message
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we validate the error message in the output of tedge sub
+Then we stop the tedge-mapper-c8y systemctl service
+
+"""
```

Comment:
I believe for this story this test is redundant unless we want to test end to end behaviour as well.

Reply:
This is kind of a black box test that tests the whole c8y_mapper. So, I feel it's better to keep it.